src/patternfly/components/Wizard/wizard.scss

```
@@ -58,6 +58,11 @@
   --#{$wizard}__nav-link-main--PaddingInlineEnd: var(--pf-t--global--spacer--sm);
   --#{$wizard}__nav-link-main--BackgroundColor: var(--pf-t--global--background--color--action--plain--default);
   --#{$wizard}__nav-link-main--BorderRadius: var(--pf-t--global--border--radius--small);
+  --#{$wizard}__nav-link-main--BorderColor: transparent;
+  --#{$wizard}__nav-link-main--BorderWidth: var(--pf-t--global--border--width--regular);
+  --#{$wizard}__nav-link--hover--nav-link-main--BorderColor: var(--pf-t--global--border--color--high-contrast);
```

Nit on the var name, and since you'll set the color by default, this can update the border width.

```suggestion
  --#{$wizard}__nav-link--hover__nav-link-main--BorderWidth: var(--pf-t--global--border--width--regular);
```